lib/concurrent/dataflow.rb

```diff
@@ -50,12 +50,15 @@ def dataflow_with!(executor, *inputs, &block)
   end
   module_function :dataflow_with!
 
-  private 
+  private
 
   def call_dataflow(method, executor, *inputs, &block)
     raise ArgumentError.new('an executor must be provided') if executor.nil?
     raise ArgumentError.new('no block given') unless block_given?
-    raise ArgumentError.new('not all dependencies are IVars') unless inputs.all? { |input| input.is_a? IVar }
+    not_ivars = inputs.map { |input| !input.is_a? IVar }
+    unless not_ivars.empty?
```

Comment:
@schneems Thank you for taking the time to work on this update. There are several failing tests we need to look into. I haven't had an opportunity to pull the code and run the tests locally, but I believe this line is the culprit. Won't the `#map` function create an array with a number of elements equal to the size of the `inputs` array? I think `#empty?` will always return `false` in this case. Can you please take a look?